core/src/main/resources/db/changelog/changesets/2023_05_23_refactor_tdr_tables.yaml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - changeSet:
+      id: "refactor_tdr_tables"
+      author: mbemis
+      changes:
+        - renameColumn:
+            oldColumnName: dataset_id
+            newColumnName: tdr_dataset_id
+            tableName: dataset
+        - addColumn:
+            tableName: data_repo_job
+            columns:
+              - column: { name: dataset_id, type: uuid, constraints: { nullable: false } }
```

Comment:
This change poses a problem for existing data because existing jobs will have null dataset IDs. I could add a raw SQL statement here to look up the dataset IDs from the dataset_name and backfill, but I think at this stage it's reasonable to just drop the data from the demo server. Production shouldn't have any datasets yet.

Reply:
Update: I just deleted the existing rows in the dev database because they were worthless, and also verified that there was nothing in prod